src/streaming/controllers/PlaybackController.js

```diff
@@ -358,7 +358,7 @@ function PlaybackController() {
             onWallclockTime();
         };
 
-        wallclockTimeIntervalId = setInterval(tick, mediaPlayerModel.getWallclockTimeUpdateInterval());
+        wallclockTimeIntervalId = wallclockTimeIntervalId || setInterval(tick, mediaPlayerModel.getWallclockTimeUpdateInterval());
```

Comment:
Do you really need to set wallclockTimeIntervalId? At the beginning of the function, there is this line : 
` if (wallclockTimeIntervalId !== null) return;`

Reply:
Removed the new one